web/src/main/java/org/mskcc/cbio/oncokb/config/MvcConfiguration.java

```diff
@@ -71,6 +102,85 @@ public ServletContextInitializer sentryServletContextInitializer() {
         return new io.sentry.spring.SentryServletContextInitializer();
     }
 
+    private ApiInfo getDefaultApiInfo(String title, String description, String apiVersion) {
+        return new ApiInfo(
+            Optional.ofNullable(title).orElse("OncoKB APIs"),
+            Optional.ofNullable(description).orElse(""),
+            Optional.ofNullable(apiVersion).orElse("v0.0.1"),
+            "https://www.oncokb.org/terms",
+            new Contact("OncoKB", "https://www.oncokb.org", "contact@oncokb.org"),
+            "Terms of Use",
+            "https://www.oncokb.org/terms"
+        );
+    }
+
+    private void updateDocketHost(Docket docket, ServletContext servletContext, String basePath) {
+        // Update swagger host/protocal with environment variable
+        String swaggerUrl = PropertiesUtils.getProperties(SWAGGER_URL);
```

Comment:
Should we add `swagger_url` to the `properties-EXAMPLE/config.properties` file? If so, we can remove the `swagger_hostname` from the file since its not used anymore.